test/ems_client_util.js

```diff
@@ -19,6 +19,7 @@
 
 
 import { EMSClient } from '../src';
+import fetch from 'node-fetch';
```

Comment:
Can we move `node-fetch` to a dev dependency now?

Reply:
yep!